config/environments/production.rb

```diff
@@ -1,75 +1,65 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
-  # Code is not reloaded between requests.
   config.cache_classes = true
 
-  # Eager load code on boot. This eager loads most of Rails and
-  # your application in memory, allowing both threaded web servers
-  # and those relying on copy on write to perform better.
-  # Rake tasks automatically ignore this option for performance.
-  config.eager_load = true
+  # Do not eager load code on boot. This avoids loading your whole application
+  # just for the purpose of running a single test. If you are using a tool that
+  # preloads Rails for running tests, you may have to set it to true.
+  config.eager_load = false
```

Comment:
Was this one of the ones that caused it to fail? Running the rails autoloader in production is a pretty significant perf penalty IIRC, and _especially_ penalizes first load (it's basically why the first request to a Rails app in dev takes up to minutes to load) so if we _can_ set it true at all, we ought to.

Reply:
I've changed these to true in qa.rb and production.rb